doc-content/enterprise-only/cases/case-management-showcase-completing-itorders-proc.adoc

```diff
@@ -15,43 +15,35 @@ image::cases/itorders-case-start.png[]
 Of these, the only user task is `Prepare hardware spec`, which is assigned to the `supplier` group. This is the first human task to be completed in the IT Orders case. When this task is complete, the `Manager approval` task becomes available to the user assigned to the `manager` role, and after the rest of the case work is finished, the `Customer satisfaction survey` task is assigned to the case owner for completion.
 
 .Prerequisite
-
-As the user `Aimee`, you have started an IT Orders case in the Showcase application with the case ID `IT-0000000001`. For more information about how to start the IT Orders case instance, see 
-ifeval::["{context}" == "case-management-showcase"]
-xref:case-management-starting-itorders-in-showcase-proc-case-management-showcase[].
-endif::[]
+As the user `wbadmin`, you have started an *IT_Orders* case in the Showcase application.
 
 .Procedure
 
-. Log out of {CENTRAL}, and log back in as a user that belongs to the `supplier` group. This example has the user `Lenovo` configured as a member of the `supplier` group.
-+
-image::cases/showcase-lenovo.png[]
+. Log out of {CENTRAL}, and log back in as a user that belongs to the `supplier` group.
 . Go to *Menu* -> *Track* -> *Task Inbox*.
 . Open the `Prepare hardware spec` task and click *Claim*. This assigns the task to the logged in user.
 . Click *Start* and click image:cases/choose-file-button.png[choose file] to locate the hardware specification file. Click image:cases/upload-button.png[the upload button] to upload the file.
 +
 image::cases/case-management-valid-spec.png[]
 +
-Click *Complete*. 
+. Click *Complete*.
 . In Showcase, click *Refresh* in the upper-right corner. Notice that the `Prepare hardware task` user task and the `Hardware spec ready` milestone appear in the *Completed* column.
 +
 image::cases/case-management-ordered.png[]
 +
-. Log out of {CENTRAL} and log back in again as `Katy`. 
-. Go to *Menu* -> *Track* -> *Task Inbox*. Open the `Manager approval` task for Katy. 
-.. Click *Claim*, then *Start*. 
+. In {CENTRAL} go to *Menu* -> *Track* -> *Task Inbox*. Open the `Manager approval` task for *wbadmin*.
+.. Click *Claim*, then *Start*.
```

Comment:
I think we have to say 
Click Claim and then click Start.

Reply:
Changed as suggested.